blueprints/simple_date_shift.py

```diff
@@ -1,19 +1,15 @@
 """
-Drop a field from the input record, or all the fields matching a blob.
+Shift a date by a random range and then shift it back to the original date with given key.
 """
 from gretel_client.transformers import DateShiftConfig
-from gretel_client.transformers import (
-    DataPath,
-    DataRestorePipeline,
-    DataTransformPipeline,
-    FieldRef,
-)
+from gretel_client.transformers import DataPath, DataRestorePipeline, DataTransformPipeline, FieldRef
```

Comment:
These are the preferred import structures, please don't revert them

Reply:
Ok sorry was not aware.